src/test/java/net/imagej/ops/AbstractOpTest.java

```diff
@@ -236,6 +240,18 @@ private int pseudoRandom() {
 		assertFalse("More elements than expected", a.hasNext());
 	}
 
+	public <T extends RealType<T>> boolean areCongruent(final IterableInterval<T> in, final RandomAccessible<T> out){
```

Comment:
That method would be super useful with an additional `epsilon` parameter to define the precision of the equality check.